Source/Revit.IFC.Import/Utility/IFCImportHandleUtil.cs

```diff
@@ -436,31 +436,5 @@ public static IList<IList<IFCAnyHandle>> GetListOfListOfInstanceAttribute(IFCAny
          }
          return outerList;
       }
-

```

Comment:
You need to keep this GetHasAssignment and all the associated calls to it because the older schema (e.g. IFC2x3) does not have the INV attribute for the HasAssignment, probably in EDM. We can probably remove this only when the ODA toolkit is ported.